testdata/cheats/Fs.t.sol

```diff
@@ -23,10 +23,10 @@ contract FsTest is DSTest {
 
         assertEq(vm.readFile(path), "hello readable world\nthis is the second line!");
 
-        vm.expectRevert(FOUNDRY_READ_ERR);
+        vm._expectCheatcodeRevert(FOUNDRY_READ_ERR);
```

Comment:
can we still use expectRevert here, or would this no longer be possible?

Reply:
We can, but the test will exit and pass right after the reverted call

That is the current behavior which resulted in some false positives